android/src/main/AndroidManifest.xml

```diff
@@ -1,8 +1,4 @@
-
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-          package="io.tradle.nfc">
-
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="io.tradle.nfc">
```

Comment:
next time please limit the changes to the specific scope of the PR, thanks! This time i'll accept :)

Reply:
Pardon me.